.changeset/yellow-feet-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `no-invalid-double-slash-comments` autofix
```

Comment:
[suggest] This PR is not related to autofix, but end positions, right?

```suggestion
Fixed: `no-invalid-double-slash-comments` end positions
```

If so, please update the PR title.

Reply:
```suggestion
Fixed: `no-invalid-double-slash-comments` reported ranges
```